showcase/src/app/demo/slider/slider.component.html

```diff
@@ -0,0 +1,133 @@
+					   [disabled]="!isSimpleHorizontalSliderEnabled"
+					   (change)="onChangeValueForSimpleHorizontalSlider($event)">
+			</mat-form-field>
+		</div>
+	</div>
+</example-viewer>
+
+<example-viewer [extensions]="['HTML', 'SCSS', 'TS']"
+				filesPath="slider/horizontal-range-slider"
+				title="SHOWCASE.DEMO.SLIDER.HORIZONTAL_RANGE_SLIDER">
+	<div>
+		<stark-slider [values]="horizontalRangeSliderValues"
+					  sliderId="horizontal-range-slider"
+					  [sliderConfig]="horizontalRangeSliderConfig"
+					  [isDisabled]="!isHorizontalRangeSliderEnabled"
+					  (changed)="onChangeHorizontalRangeSlider($event)">
+		</stark-slider>
+		<div>
+			<mat-checkbox class="slider-enabled-field"
+						  (change)="$event ? toggleHorizontalRangeSliderEnable() : null" 
+						  [checked]="isHorizontalRangeSliderEnabled">
+				{{ 'SHOWCASE.COMMON.ENABLED' | translate }}
+			</mat-checkbox>	
+			<mat-form-field class="slider-value-field">
+				<input matInput 
+					   placeholder="{{ 'SHOWCASE.COMMON.VALUE' | translate }} 1" 
+					   type="number"
+					   step="10"
+					   [(ngModel)]="lowerValueForHorizontalRangeSlider"
+					   [disabled]="!isHorizontalRangeSliderEnabled"
+					   (change)="onChangeValueForHorizontalRangeSlider($event, SLIDER_HANDLES.lower)">
+			</mat-form-field>
+			<mat-form-field class="slider-value-field">
+				<input matInput
+					   placeholder="{{ 'SHOWCASE.COMMON.VALUE' | translate }} 2"
+					   type="number"
+					   step="10"
+					   [(ngModel)]="upperValueForHorizontalRangeSlider"
+					   [disabled]="!isHorizontalRangeSliderEnabled"
+					   (change)="onChangeValueForHorizontalRangeSlider($event, SLIDER_HANDLES.upper)">
+			</mat-form-field>
+		</div>
+	</div>
+</example-viewer>
+
+<example-viewer [extensions]="['HTML', 'SCSS', 'TS']"
+				filesPath="slider/simple-vertical-slider"
+				title="SHOWCASE.DEMO.SLIDER.SIMPLE_VERTICAL_SLIDER">
+	<div>
+		<stark-slider [values]="simpleVerticalSliderValues"
+					  sliderId="simple-vertical-slider"
+					  [sliderConfig]="simpleVerticalSliderConfig"
+					  [isDisabled]="!isSimpleVerticalSliderEnabled"
+					  (changed)="onChangeSimpleVerticalSlider($event)">
+		</stark-slider>
+		<div>
+			<mat-checkbox class="slider-enabled-field"
+						  (change)="$event ? toggleSimpleVerticalSliderEnable() : null"
+						  [checked]="isSimpleVerticalSliderEnabled">
+				{{ 'SHOWCASE.COMMON.ENABLED' | translate }}
+			</mat-checkbox>
+			<mat-form-field class="slider-value-field">
+				<input matInput
+					   placeholder="{{ 'SHOWCASE.COMMON.VALUE' | translate }}"
+					   type="number"
+					   step="10"
+					   [(ngModel)]="valueForSimpleVerticalSlider"
+					   [disabled]="!isSimpleVerticalSliderEnabled"
+					   (change)="onChangeValueForSimpleVerticalSlider($event)">
+			</mat-form-field>
+		</div>
+	</div>
+</example-viewer>
+
+<example-viewer [extensions]="['HTML', 'SCSS', 'TS']"
+				filesPath="slider/vertical-range-slider"
+				title="SHOWCASE.DEMO.SLIDER.VERTICAL_RANGE_SLIDER">
```

Comment:
Same here, this binding has been changed after your PR.
Could you please adapt this for `exampleTitle="SHOWCASE.DEMO.SLIDER.VERTICAL_RANGE_SLIDER"` ?